glide.yaml

```diff
@@ -3,33 +3,3 @@ import:
 - package: github.com/Azure/acs-engine
   version: osaapi
   repo:    https://github.com/jim-minter/acs-engine
-  subpackages:
-  - pkg/api
-  - pkg/osa/vlabs
-- package: github.com/ghodss/yaml
-- package: github.com/satori/uuid
-- package: golang.org/x/crypto
-  subpackages:
-  - bcrypt
-  - ssh
-- package: k8s.io/apimachinery
-  subpackages:
-  - pkg/api/errors
-  - pkg/api/meta
-  - pkg/apis/meta/v1
-  - pkg/apis/meta/v1/unstructured
-  - pkg/runtime/schema
-  - pkg/util/wait
-- package: k8s.io/client-go
```

Comment:
Why are these removed?

Reply:
They're all transitively implied by the acs-engine import, which is currently pinning a lot of versions.  That's a pretty horrible situation and this removal helps to make that evident.  We'll want to somehow cut that out so we can decide the versions of the libraries we need, not have it decided for us on the basis of importing a single API package :-/

Reply:
That's pretty bad, we use client-go directly in `sync` so it's not a transitive dependency and certain client-go versions map to certain k8s versions.

Reply:
What about if we kick-off separate vendor tree/sub project for projects not dependent on acs-engine (like sync)?